### ISSUE TITLE
App Randomly Closes and Video Continues Playing In Background

### ISSUE THREAD

**Issue by CybearWarfare** [Nanoc6/SaikouTV#15]
On multiple occasions on my Nvidia Shield I have had the app close while the video continues to play in the background and the only way to stop it is restarting the device or waiting for it to finish playing.

**Comment by Nanoc6** [comment]
Yeah i think i saw that happening at least once, ill look into it, thanks.

**Comment by venkat2305** [comment]
I faced this issue.

**Comment by anime-src** [comment]
Same, had to go back into player, press pause :) and back to android tv home screen

**Comment by roopaish** [comment]
The same thing happened to me. Additionally, the video does not pause while it's loading. Often, I exit out of the app while it's loading, and it continues to run in the background. Even if I open the app and play a new video, the old video continues to play in the background
